mgmtsystem/azure.py

```diff
@@ -20,6 +20,7 @@
 class AzureSystem(MgmtSystemAPIBase):
     """This class is used to connect to Microsoft Azure Portal via PowerShell AzureRM Module
     """
+    DEFAULT_RESOURCE_GROUP = "CFMEQE-Main"
```

Comment:
@jteehan and I spoke about this, we will be moving this to the yaml and setting it at instantiation time as a default to be used it no resourcegroup is supplied to the below fns